debezium-core/src/main/java/io/debezium/util/Strings.java

```diff
@@ -909,6 +909,15 @@ public static String unquoteIdentifierPart(String identifierPart) {
         return bytes;
     }
 
+    public static String rightPad(String input, int length, char c) {
```

Comment:
There's already `Strings#justifyLeft()`. I'll change it to use that one instead.